content/getting-started/device-farm-tutorial.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Use Alexa App on Device Farm (optional)"
+date: 2019-09-26T13:29:22+08:00
+draft: false
+chapter: false
```

Comment:
Have you tried to run this locally? need `weight` parameter I think.

Reply:
Yes, and I think `weight` is not necessary.

Reply:
If you do not include `weight` parameter, then the left side navigation bar will rank the page according to the alphabet.. Which is not what we wanted. Put this article under [BIND ECHO DEVICE (OPTIONAL)]

Reply:
Done.